article-recommendations-widget.php

```diff
@@ -146,34 +230,48 @@ function lnl_get_recommendations($results) {
 		 * @return void
 		 */
 		function lnl_get_recent_posts() {
+
 			$args = array( 'numberposts' => '5' );
 			$recent_posts = wp_get_recent_posts($args);
+			$model_type = 'recent_posts'; // model context
+
+			$one = $recent_posts[0]['ID'];
+			$two=$recent_posts[1]['ID'];
+			$three=$recent_posts[2]['ID'];
+			$four=$recent_posts[3]['ID'];
+			$five=$recent_posts[4]['ID'];
+			$str = "data-vars-one=$one ";
+			$str .= "data-vars-two=$two ";
+			$str .= "data-vars-three=$three ";
+			$str .= "data-vars-four=$four ";
+			$str .= "data-vars-five=$five ";
+			$str .= "data-vars-model-type='recent_posts' ";
```

Comment:
should we interpolate `model_type` into the string, rather than hardcoding recent_posts?